amundsen_application/static/js/config/config-utils.ts

```diff
@@ -205,3 +205,41 @@ export function generateExploreUrl(tableData: TableMetadata): string {
 export function getMaxLength(key: string) {
   return AppConfig.editableText[key];
 }
+
+/**
+ * Returns the display name for a given description source id for a given resource type.
+ * If a configuration or display name does not exist for the given description source id, the id
+ * is returned.
+ */
+export function getDescriptionSourceDisplayName(sourceId: string): string {
+  const config = AppConfig.resourceConfig[ResourceType.table];
+  if (
+    !config ||
+    !config.supportedDescriptionSources ||
+    !config.supportedDescriptionSources[sourceId] ||
+    !config.supportedDescriptionSources[sourceId].displayName
```

Comment:
Nit: usually it is easier to understand a non-negated condition, so it would be easier to do:
 ``` if (
    config ||
    config.supportedDescriptionSources ||
    config.supportedDescriptionSources[sourceId] ||
    config.supportedDescriptionSources[sourceId].displayName
){
 return config.supported....
}
return sourceId;```

Reply:
👍